cluster/ci/config/prow/plugins.yaml

```diff
@@ -329,6 +329,12 @@ plugins:
   openshift/cluster-autoscaler-operator:
   - approve
 
+  openshift/ansible-service-broker:
+  - trigger
```

Comment:
this is already enabled in [here](https://github.com/openshift/release/blob/master/cluster/ci/config/prow/plugins.yaml#L19)